From df44355beba5a89987a4a5cad0517bfc43061706 Mon Sep 17 00:00:00 2001
From: Bing Xu <bing.xu@intel.com>
Date: Thu, 27 Jan 2022 10:24:56 +0800
Subject: [PATCH] Delete some opts not supported on common kernel.

Some test case of StorageManagerTest will fail as
we use common kerel and don't support some fuse mount
opts, such as context and fscontext.

Change-Id: Ie40ebf4d77b7fa027fbd0a16e8f1dd801cba39ec
Signed-off-by: Bing Xu <bing.xu@intel.com>
---
 AppFuseUtil.cpp | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/AppFuseUtil.cpp b/AppFuseUtil.cpp
index 711e70b5..995326d0 100644
--- a/AppFuseUtil.cpp
+++ b/AppFuseUtil.cpp
@@ -54,9 +54,7 @@ static android::status_t Mount(int device_fd, const std::string& path) {
         "rootmode=40000,"
         "default_permissions,"
         "allow_other,"
-        "user_id=0,group_id=0,"
-        "context=\"u:object_r:app_fuse_file:s0\","
-        "fscontext=u:object_r:app_fusefs:s0",
+        "user_id=0,group_id=0,",
         device_fd);
 
     const int result =
-- 
2.17.1

